src/applications/simple-forms/40-0247/helpers.js

```diff
@@ -10,12 +10,14 @@ import {
   waitForRenderThenFocus,
 } from 'platform/utilities/ui';
 import scrollTo from 'platform/utilities/ui/scrollTo';
+import environment from 'platform/utilities/environment';
 
 export function trackNoAuthStartLinkClick() {
   recordEvent({ event: 'no-login-start-form' });
 }
 
-export function getInitialData({ mockData, environment }) {
+// environment inside the getInitialData is removed for now until the prod flag on line 96 is removed
```

Comment:
there should be a way to test exactly what you want without changing any of the other code. you can use `sinon.stub` to force environment to return whatever you want for your specific tests

e.g.
```js
const environmentStub = sinon.stub(environment, 'isProduction').returns(true);
... tests ...
environmentStub.restore()
```

Reply:
Hmm ok gotcha. I'm not sure I need this in this current situation but I could be wrong? Please tell me more if you'd like and I can go in and make a change. The issue is really just the use of the word 'environment' when it comes to using our prod flags.